app/build.gradle

```diff
@@ -8,7 +8,6 @@ apply plugin: 'io.fabric'
 
 android {
     compileSdkVersion 27
-    buildToolsVersion '27.0.3'
```

Comment:
Didn't know this guy is optional. I'm not sure should it be deleted or not. On the first glance, I would rather want it back because Android Studio will highlight it if there is a newer version. Also, it seems like it's better to have the same version of build tools for every developer who works on the project.

Reply:
With the Android Plugin 3.0.0+, it can infer which build tools it needs based on the plugin version. https://developer.android.com/studio/releases/build-tools You can also specify it manually though. I can go ahead and change it back though.

Reply:
Aha, got it. No need to change it back.